### PR TITLE
docs: add example for template input variable

### PR DESCRIPTION
**Pull request by abadakhshan**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: https://github.com/angular/angular/issues/43070

## What is the new behavior?

missing example added to the docs.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/guide/template-reference-variables.md [hunk 1]
```diff
@@ -162,6 +162,12 @@ You declare a template input variable using the `let` keyword as in `let hero`.
 
 There are several such variables in this example: `hero`, `i`, and `odd`.
 
+```html
+<ng-template #hero let-hero let-i="index" let-odd="isOdd">
+  <div [ngClass]="{'odd-row': odd}">{{i}}:{{hero.name}}</div>
```
Comment (AndrewKushnir) [inline aio/content/guide/template-reference-variables.md:167]: I'd propose using `[class]` binding instead (so there is no need to use an extra directive):
```suggestion
  <div [class]="{'odd-row': odd}">{{i}}:{{hero.name}}</div>
```

**Review by AndrewKushnir — APPROVED** [review]
@abadakhshan thanks for improving Angular docs! I've just added a minor comment. Could you please have a look when you get a chance? Thank you.

**Comment by abadakhshan** [comment]
> @abadakhshan thanks for improving Angular docs! I've just added a minor comment. Could you please have a look when you get a chance? Thank you.

Thank you @AndrewKushnir, I did it

**Comment by AndrewKushnir** [comment]
@abadakhshan thanks for the update. Could you please squash 2 commits into one and leave a comment here once it's done, so we can add this PR to the merge queue. Thank you.

**Comment by abadakhshan** [comment]
I'm so sorry @AndrewKushnir.
I did something wrong and this PR was closed.

I created another  one: https://github.com/angular/angular/pull/43137

**Pull request closed without merging**